src/components/Header/Header.js

```diff
@@ -18,16 +18,16 @@ function Header()
                     src={Logo}
                     alt="Linguiçator Logo"
                 />
-                <h1>Linguiçator</h1>
+                <h1>Lingüiçator</h1>
```

Comment:
Just to be sure, this change was intentional? Since it does not fit the commit description I thought it might have been accidental. Nothing against the change btw, just confirming.

Reply:
Yes, as I said earlier, `Trema na linguiça!`